From 5b732409aebd2bcf1954d1d454f610ba2c0c9b1e Mon Sep 17 00:00:00 2001
From: Xuefu Zhang <xuefu@uber.com>
Date: Thu, 23 Mar 2017 19:36:16 -0700
Subject: [PATCH 1048/1164] CDH-51969: HIVE-16286: Log canceled query id
 (Jimmy via Xuefu)

(cherry picked from commit 8fa9d5833c5c4fda092a917c9881db8fbfea5332)

Change-Id: I219e8e49cfad75ea2838d0d3beb86ebf4012d267
---
 .../hive/service/cli/operation/SQLOperation.java   |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/service/src/java/org/apache/hive/service/cli/operation/SQLOperation.java b/service/src/java/org/apache/hive/service/cli/operation/SQLOperation.java
index be9ad53..767ecc7 100644
--- a/service/src/java/org/apache/hive/service/cli/operation/SQLOperation.java
+++ b/service/src/java/org/apache/hive/service/cli/operation/SQLOperation.java
@@ -157,8 +157,9 @@ public void prepare(HiveConf sqlOperationConf) throws HiveSQLException {
           @Override
           public void run() {
             try {
+              String queryId = confOverlay.get(HiveConf.ConfVars.HIVEQUERYID.varname);
               LOG.info("Query timed out after: " + queryTimeout
-                  + " seconds. Cancelling the execution now.");
+                  + " seconds. Cancelling the execution now: " + queryId);
               SQLOperation.this.cancel(OperationState.TIMEDOUT);
             } catch (HiveSQLException e) {
               LOG.error("Error cancelling the query after timeout: " + queryTimeout + " seconds", e);
-- 
1.7.9.5

